pkg/releasetesting/test_suite_test.go

```diff
@@ -220,6 +220,38 @@ func TestExtractTestManifestsFromHooks(t *testing.T) {
 	}
 }
 
+func TestParallelTestRun(t *testing.T) {
```

Comment:
Worth adding a test in which we execute tests in parallel and one test fails? Your choice if thats necessary :)

Reply:
imo that's not necessary, as there's no new behaviour on passing/failing tests - I could be convinced otherwise though

Reply:
I think it's worth testing all possible real world scenarios. We've definitely hit regressions before by missing scenarios like this one and finding regressions several releases down the road when other contributors start extending the feature into other use cases.

I know it puts a little bit more effort on yourself to add these tests, but we're really striving for quality over quantity when it comes to new features for Helm. Thanks :heart: